packages/components/Atoms/Global/scss/settings/_button.scss

```diff
@@ -15,7 +15,7 @@ $rpl-button-secondary-text-color: rpl_color('white') !default;
 $rpl-button-secondary-hover-background-color: rpl_color('primary') !default;
 $rpl-button-secondary-hover-text-color: $rpl-button-secondary-text-color !default;
 $rpl-button-danger-background-color: rpl_color('danger') !default;
-$rpl-button-quick-exit-hover-background-color: lighten(rpl-color('danger'), 5%) !default;
```

Comment:
It will be dangerous to remove an existing variable as it will cause breaking change when some customer projects use it. We have to make sure it's safe to do otherwise we'd rather keep it and deprecate it. 
However I have did a global search for this one in all projects and it's safe.

Reply:
Oh I see, will keep that in mind next time.